roles/config-httpd/README.md

```diff
@@ -0,0 +1,55 @@
+Set of Roles
+============
+
+The ansible roles found in this directory are associated with configuring a httpd web server. It is broken into two steps, prep and seeding. In prep it sets up httpd and firewalld (opening it up on port 80). In seed it fills up the web server with content from a pre-determined location.
+
+Requirements
+------------
+
+No specific system requirements.
+
+Role Variables
+--------------
+httpd_seed_dir sets up the directory for where to retrieve the content, eventually to be used to seed the web server.default_document_root merely sets the default root folder for the where to deposit the web files that have been retrieved.
+
+```
+html_document_root: "/var/www/html"
```

Comment:
Make sure to call out which ones of these are required v.s. optional. Also, in general, instead of the "description" above, I'd recommend just using a table to outline these variables - i.e.: something like what's found here: 

https://github.com/redhat-cop/infra-ansible/tree/master/roles/manage-jira#role-variables